backend/src/test/circuit-router.spec.js

```diff
@@ -0,0 +1,33 @@
+import request from "supertest";
```

Comment:
These arent really unit tests since they test a whole code path. Can you move these test to the `backend/src/test/index.spec.js` file since that one can contain all of our API endpoint tests